.github/build.main.yml

```diff
@@ -4,95 +4,238 @@ trigger:
+            Contents: "**/*.nupkg"
+            TargetFolder: "$(Build.ArtifactStagingDirectory)"
+            CleanTargetFolder: true
+            OverWrite: true
+            flattenFolders: true
+
+        - task: PublishBuildArtifacts@1
+          displayName: "save artifacts"
+          inputs:
+            ArtifactName: packages
+
+  - stage: Python
+    condition: succeeded()
+    jobs:
+    - job: Test
+      variables:
+        PythonWrapperBaseDirectory: "$(Build.SourcesDirectory)/wraps/python"
+        DotNetVersion: "netstandard2.0"
+
+      strategy:
+        matrix:
+          Windows-Python38:
+            imageName: windows-latest
+            python.version: '3.8'
+          Windows-Python39:
+            imageName: windows-latest
+            python.version: '3.9'
+          Linux-Python38:
+            imageName: ubuntu-latest
+            python.version: '3.8'
+          Linux-Python39:
+            imageName: ubuntu-latest
+            python.version: '3.9'
+
+      pool:
+        name: Azure Pipelines
+        vmImage: '$(imageName)'
+
+      steps:
+      - task: UsePythonVersion@0
+        inputs:
+          versionSpec: '$(python.version)'
+
+      - task: DownloadPipelineArtifact@2
+        displayName: "Fetch NuGet from artifacts"
+        inputs:
+          artifact: packages
+          patterns: 'packages/*.nupkg'
+
+      - task: ExtractFiles@1
+        displayName: "Extract DLL from NuGet"
+        inputs:
+          archiveFilePatterns: '$(Pipeline.Workspace)/*.nupkg'
+          destinationFolder: '$(Build.SourcesDirectory)/unpacked'
+          cleanDestinationFolder: true 
+          overwriteExistingFiles: true
+
+      - task: CopyFiles@2
+        displayName: "Replace DLL"
+        inputs:
+          SourceFolder: "$(Build.SourcesDirectory)/unpacked/lib/$(DotNetVersion)"
+          Contents: "*.dll"
+          TargetFolder: "$(PythonWrapperBaseDirectory)/SkenderStockIndicators/_cslib/lib"
+          CleanTargetFolder: true
+          OverWrite: true
+          flattenFolders: true
+
+      - script: |
+          python -m pip install --upgrade pip setuptools wheel
+          pip install pythonnet
+        displayName: 'Install dependencies'
+
+      - script: |
+          pip install pytest pytest-cov openpyxl
+          pytest -vr A --junitxml=$(PythonWrapperBaseDirectory)/tests/test-results.xml --cov=$(PythonWrapperBaseDirectory)/SkenderStockIndicators --cov-report xml:$(PythonWrapperBaseDirectory)/tests/coverage.xml
+        displayName: 'Run all tests'
+
+      - task: PublishTestResults@2
+        condition: succeededOrFailed()
+        inputs:
+          testResultsFiles: '$(PythonWrapperBaseDirectory)/tests/test-results.xml'
+          testRunTitle: 'Python $(python.version) Tests'
+
+    - job: Build
+      variables:
+        PythonWrapperBaseDirectory: "$(Build.SourcesDirectory)/wraps/python"
+        ImageName: "ubuntu-latest"
+        DotNetVersion: "netstandard2.0"
+        PythonVersion: "3.8"
```

Comment:
When we're building this package for PyPI, does it matter that we're only using one version here?  Or is there an option or need to also build for v3.9?  I'm unclear on how PyPI works in this regard.

Reply:
I think the version of Python doesn't matter when we build. But to make it be sure, I posted about it on Stack Overflow. 
 * https://stackoverflow.com/questions/67761075/which-python-version-to-be-used-for-building-pypi-package

Reply:
I found a problem about minimum Python version of our package. It seems like that `pythonnet` supports Python 3.5 and higher. But when I tested on Azure Pipelines, errors occurred when Python 3.5~3.7. I just thought that versions of Python are not supported, but it was wrong. So we need to figure out. I think it's better to deal with this problem on new Github issue for now.

Reply:
Yah, go ahead and create an issue to track with mention of this conversation.  We have some time before we're publishing, but will need to figure it out along the way.  No need to hold this PR up.